source/clear-linux/get-started/virtual-machine-install/vmware-player-2.rst

```diff
@@ -0,0 +1,469 @@
+.. _vmware-player:
+
+Use VMware\* Workstation Player
+###############################
+
+This section explains how to run |CLOSIA| as a virtual machine using the 
+`VMware Workstation 14 Player`_ hypervisor.
+
+There are 2 ways to create a |CL| VM to run in VMware:
+
+* `Method #1`: Fresh installation of |CL| into new VM.  This provides flexibility 
+  in configuring the VM size, partitions, initial |CL| bundles selection, etc.
+* `Method #2`: Use a ready-made VMware |CL| image with minimally required bundles.  
+
+Both are discussed below.  
+
+.. note::
+
+  The figures shown throughout this document are from the Windows version of 
+  VMware Workstation 14 Player.  They are applicable for the Linux version as
+  well (with minor differences).
```

Comment:
@bktan8 are the differences just in style/design/layout? Same descriptions to be checked, etc?